cpp/lib/ome/bioformats/tiff/IFD.cpp

```diff
@@ -1057,7 +1090,11 @@ namespace ome
             break;
 
           default:
-            throw Exception("Unsupported OME data model PixelType");
+            {
+              boost::format fmt("Unsupported OME data model PixelType %1%");
```

Comment:
Looked through a bit more carefully.  They are subtly different.  This one maps pixeltype to sampleformat/bit depth.  It's constrained to the pixeltype enum.  The other one is the reverse mapping and is not constrained; a TIFF could potentially have sampleformats and bit depth combinations we simply can't handle, which is why this is "unsupported by the data model", rather than "unsupported pixel type"--there isn't a pixel type we can map to.